packages/shared/src/dto/subscriber/list-subscribers.dto.ts

```diff
@@ -10,7 +10,7 @@ export interface IListSubscribersRequestDto {
 
   orderDirection: DirectionEnum;
 
-  orderBy: 'updatedAt' | 'createdAt';
+  orderBy: 'updatedAt' | '_id';
```

Comment:
as mentioned above IMO we should encapsulate internal implementation on the API interface, the client should sort `createdAt` and we will map it to sort by `_id`